sklearn/ensemble/_forest.py

```diff
@@ -1475,17 +1499,22 @@ class RandomForestRegressor(ForestRegressor):
         the input samples) required to be at a leaf node. Samples have
         equal weight when sample_weight is not provided.
 
-    max_features : {"auto", "sqrt", "log2"}, int or float, default="auto"
+    max_features : {"sqrt", "log2"}, int or float, default=1.0
         The number of features to consider when looking for the best split:
 
         - If int, then consider `max_features` features at each split.
         - If float, then `max_features` is a fraction and
           `round(max_features * n_features)` features are considered at each
           split.
+        - The default of 1.0 is equivalent to bagged trees and more
+          randomness can be achieved by setting smaller values e.g. 0.3.
         - If "auto", then `max_features=n_features`.
-        - If "sqrt", then `max_features=sqrt(n_features)`.
+        - If "sqrt", then `max_features=sqrt(n_features)` (same as "auto").
```

Comment:
```suggestion
```

Reply:
Hi @lorentzenchr , are we sure about this change? It would completely remove what `max_features="sqrt"` does from this particular docstring - what if the user is unsure what that setting does?

Reply:
I think that @lorentzenchr wanted to remove `(same as "auto")` only since we are deprecating this option.

Reply:
Yes, only the part "(same as auto)" as it is wrong.